kahoot-flood/main.go

```diff
@@ -29,7 +29,7 @@ func main() {
 	for i := 0; i < ConcurrencyCount; i++ {
 		go func() {
 			for {
-				conn, err := kahoot.NewConn(gamePin)
+				conn, err := kahoot.NewConn(785071)
```

Comment:
This is unneeded, this ignores the set pin

Reply:
thanks